nautobot/docs/additional-features/jobs.md

```diff
@@ -576,6 +576,9 @@ class MyJobTestCase(TransactionTestCase):
         # Testing of Job "MyJob" in file "my_job_file.py" in $JOBS_ROOT
         job = Job.objects.get(job_class_name="MyJob", module_name="my_job_file", source="local")
         # or, job = Job.objects.get_for_class_path("local/my_job_file/MyJob")
+        # As tests run on a clean database each time, we need to make sure the job is enabled
```

Comment:
Two ideas:

1) Could call this out explicitly in the text as well as in the code sample?
2) What if we enhanced `run_job_for_testing` so that it automatically enables the job before running it, so that this code wouldn't be necessary?

Reply:
Idea two is implemented. For the tests under `nautobot.extras.tests.test_jobs.RunJobManagementCommandTest` I went through and "manually" enabled the tests for now, I wanted to ask though if I should abstract this out into `RunJobManagementCommandTest.run_command`?

As for idea one - by this you mean putting a bit of prose that mentions we have to enable jobs but `run_job_for_testing` handles this implicitly?

Reply:
> As for idea one - by this you mean putting a bit of prose that mentions we have to enable jobs but `run_job_for_testing` handles this implicitly?

Yes, exactly. :-)